plugin/mutate/source/dextool/plugin/mutate/backend/database/schema.d

```diff
@@ -411,6 +412,30 @@ struct MutantTimeoutCtx {
     State state;
 }
 
+import dextool.plugin.mutate.backend.database.type : Rationale;
+
+@TableName(markedMutantTable)
+@TablePrimaryKey("mutationStatusId")
+struct MarkedMutant {
+    @ColumnName("st_id")
+    ulong mutationStatusId;
+
+    uint line;
+
+    uint column;
+
+    string path;
+
+    @ColumnName("status")
```

Comment:
Might as well change the name of `to_status` to status which would make this UDA redundant